### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by lienht-1767**
=> Title: admin
=> Redmine: https://edu-redmine.sun-asterisk.vn/issues/11081
=> Attachment File:
Screenshot from 2019-04-22 16-49-20
Screenshot from 2019-04-22 16-49-39
Screenshot from 2019-04-22 16-49-59
Screenshot from 2019-04-22 16-50-15
Screenshot from 2019-04-22 16-50-31

### CODE REVIEW

**Comment by lienht-1767** [comment]
ready

File: app/controllers/admin/genders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      flash[:success] = t(".updated")
+      redirect_to admin_genders_path
+    else
+      flash[:error] = t(".fail")
+      render :edit
+    end
+  end
+
+  def destroy
+    @a = Gender.includes(:information_user)
```
Comment (huuphuc93) [inline app/controllers/admin/genders_controller.rb]: sao lại đặt tên biến như này em

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class Admin::UsersController < Admin::BaseController
+  before_action :load_user, only: %i(show destroy)
+
+  def index
+    @users = User.page(params[:page]).per Settings.per_page
+  end
+
+  def show
+    @info = InformationUser.find_by user_id: params[:id]
```
Comment (huuphuc93) [inline app/controllers/admin/users_controller.rb:9]: check đk k tìm thấy @info

File: app/views/admin/genders/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+              <th><%= t ".created_at" %></th>
+              <th colspan="2"><%= t ".action" %></th>
+            </tr>
+          </thead>
+          <tbody>
+            <% @genders.each_with_index do |gender,index| %>
+            <tr>
+              <td><%= create_index params[:page], index, Settings.per_page %></td>
+              <td><%= gender.name %></td>
+              <td><%= gender.created_at.to_time.strftime t(".format_time") %></td>
```
Comment (huuphuc93) [inline app/views/admin/genders/index.html.erb]: I18n cho đoạn time này

File: app/views/admin/users/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+              <th><%= t ".name" %></th>
+              <th><%= t ".created_at" %></th>
+              <th colspan="2"><%= t ".action" %></th>
+            </tr>
+          </thead>
+          <tbody>
+            <% @users.each_with_index do |user,index| %>
+            <tr>
+              <td><%= create_index params[:page], index, Settings.per_page %></td>
+              <td><%= user.information_user.first_name + " " +
```
Comment (huuphuc93) [inline app/views/admin/users/index.html.erb:19]: check n + 1 nhé

**Comment by lienht-1767** [comment]
ready

**Pull request merged**